applications/blackhole/src/bh_context.erl

```diff
@@ -197,16 +170,38 @@ set_account_id(#bh_context{}=Context, AcctId) ->
 %% @end
 %%--------------------------------------------------------------------
 -spec bindings(context()) -> ne_binaries().
-bindings(#bh_context{bindings=Bindings}) ->
-    Bindings.
+bindings(#bh_context{bindings=Bds}) ->
+    Bds.
+
+-spec bindings_from_json(wh_json:object()) -> ne_binary() | ne_binaries().
+bindings_from_json(JObj) ->
+    wh_json:get_first_defined([<<"binding">>, <<"bindings">>], JObj).
```

Comment:
Could we only support `bindings`? Allowing both when the difference is that one takes an array of what the other takes, I find that very weird.